src/cloud/docker/docker-containers-service.md

```diff
@@ -0,0 +1,176 @@
+---
+group: cloud-guide
+title: Docker Services Containers
+functional_areas:
+  - Cloud
+  - Docker
+  - Configuration
+---
+
+## Service Containers
+
+The following containers provide the services required to build, deploy and run Magento 2 sites.
+
+{:.bs-callout-info}
+See the [service version values available]({{ site.baseurl }}/cloud/docker/docker-containers.html#service-versions) for use when launching Docker.
+
+### Database Container
+
+#### Container Information
+
+-  Name: db
+-  Docker Base Image: [mariadb](https://hub.docker.com/_/mariadb)
+-  Ports Exposed 3306
+
+#### Container Usage
+
+To import a database dump, place the SQL file into the `.docker/mysql/docker-entrypoint-initdb.d` folder.
+
+> When a container is started for the first time, a new database with the specified name will be created and initialized with the provided configuration variables. Furthermore, it will execute files with extensions .sh, .sql and .sql.gz that are found in /docker-entrypoint-initdb.d. Files will be executed in alphabetical order.
```

Comment:
```suggestion
When a container is started for the first time, a new database with the specified name is created and initialized with the provided configuration variables. Furthermore, it will execute files with extensions .sh, .sql and .sql.gz that are found in the `/docker-entrypoint-initdb.d` directory. Files will be executed in alphabetical order.
```